lib/components/Tag/Tag.docs.tsx

```diff
@@ -46,6 +46,30 @@ const docs: ComponentDocs = {
       ),
     },
   ],
+  snippets: [
+    {
+      name: 'Standard',
+      code: <Tag>Tag</Tag>,
```

Comment:
Should these always be nested in an `Inline` like badges? Will they always be a set?